1-3.md

```diff
@@ -125,32 +131,30 @@ Introduce other benefits of accessibility, such as technical or financial. Use t
 
 {% include excol.html type="middle" %}
 
-Emphasize on success cases from organizations that have implemented accessibility into their products. Underline how that contributes to avoiding legal risks. Use the resource [The Business Case for Digital Accessibility (Minimize Legal Risk)](https://www.w3.org/WAI/business-case/#minimize-legal-risk).
```

Comment:
Again, very brief and unclear IMO. What are "success cases" and are there any for the different countries around the world? Need to be more balanced for the many countries without (effective) policies